modules/lx-autoscale/watchmaker-lx-autoscale.template.cfn.yaml

```diff
@@ -610,6 +618,49 @@ Resources:
                         "log_stream_name": "default_logs_{instance_id}"
                       }
                     }
+            /etc/cfn/scripts/aws-cw-agent/cloudwatch-applog-config.py:
+              content: !Sub
+                - |-
+                  import json
+                  import os
+                  
+                  cloudwatch_baseline = "/opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.json"
+                  log_group_name = "/aws/ec2/lx/${AWS::StackName}"
+                  log_paths_input = ('''${local_addCWLtarget}''').split(",")
+                  log_objs = []
+
+
+                  def get_stream_name(log_path):
+                      return "{}_logs_{{instance_id}}".format(os.path.basename(log_path))
+
+
+                  def load_baseline():
+                      print('Loading existing cwl baseline')
+                      with open(cloudwatch_baseline, 'r') as f:
+                          return json.load(f)
+
+
+                  def write_baseline(baseline):
+                      print('Writing new cwl baseline')
+                      with open(cloudwatch_baseline, 'w') as f:
+                          f.write(json.dumps(baseline, sort_keys=True, indent=4))
+
+
+                  if log_paths_input:
+                      print('Additional cwl paths were defined.')
+                      baseline = load_baseline()
+                      for log_path in log_paths_input:
+                          log_path = log_path.rstrip(',')
```

Comment:
since we split on commas, it should no longer be possible for any of these paths to end in a comma. such a construction of the comma-separated list would result in an empty item instead of a trailing comma, so pretty sure this line is extraneous with the current approach.